docs/UsersGuide/source/WE2Etests.rst

```diff
@@ -3,120 +3,445 @@
 ================================
 Workflow End-to-End (WE2E) Tests
 ================================
-The SRW Application's experiment generation system contains a set of end-to-end tests that 
-exercise various configurations of that system as well as those of the pre-processing, 
-UFS Weather Model, and UPP post-processing codes. The script to run these tests is named 
-``run_experiments.sh`` and is located in the directory ``ufs-srweather-app/regional_workflow/tests``.
-A complete list of the available tests can be found in ``baselines_list.txt`` in that directory.   
-This list is extensive; it is not recommended to run all of the tests as some are computationally 
-expensive.  A subset of the tests supported for this release of the SRW Application can be found 
-in the file ``testlist.release_public_v1.txt``. 
- 
-The base experiment configuration file for each test is located in the ``baseline_configs`` 
-subdirectory.  Each file is named ``config.${expt_name}.sh``, where ``${expt_name}`` is the 
-name of the corresponding test configuration. These base configuration files are subsets of
+The SRW App contains a set of end-to-end tests that exercise the App in various configurations.
+These are referred to as workflow end-to-end (WE2E) tests because they all use the Rocoto 
+workflow manager to run their individual workflows.  
+The purpose of these tests is to 
+ensure that new changes to the App do not break existing functionality and capabilities.  
+
+
+Note that the WE2E tests are not regression tests, i.e. they do not check whether 
+current results are identical to previously established baselines.  They are also
+not tests of the scientific integrity of the results, e.g. they do not check that values 
+of output fields are reasonable.  
+These tests only check that the tasks within each test's workflow complete
+successfully.  
+They are in essence tests of the workflow generation, task execution (j-jobs, 
+ex-scripts), and other auxiliary scripts (which are mostly in the ``regional_workflow``
+repo) to ensure that these scripts are correctly performing their functions.  These functions
+include creating and correctly arranging and naming directories and files, ensuring 
+that all input files are available and readable, calling
+executables with correct namelists and/or options, etc.
+For the time being, it is up to the external repositories that the App clones (:numref:`Section %s <SRWStructure>`)
+to check that changes to those repos do not change results, or, if they do, to ensure that the new
+results are acceptable.  
+(At least two of these external repositories --- ``UFS_UTILS`` and ``ufs-weather-model`` ---
+do have such regression tests.)  
```

Comment:
@jkbk2004 @EdwardSnyder-NOAA Aren't you guys working on SRW App regression tests? Can we add anything here?

Reply:
I have been working on WE2E testing for the SRW App, but I still read over the changes and see nothing to add.